src/settings/MappingProfiles/detailsSections/view/OrderDetailsSection/OrderDetails.js

```diff
@@ -0,0 +1,256 @@
+  const subscriptionFrom = getFieldValue(mappingDetails, 'subscriptionFrom', 'value');
+  const subscriptionTo = getFieldValue(mappingDetails, 'subscriptionTo', 'value');
+  const subscriptionInterval = getFieldValue(mappingDetails, 'subscriptionInterval', 'value');
+  const publicationDate = getFieldValue(mappingDetails, 'publicationDate', 'value');
+  const publisher = getFieldValue(mappingDetails, 'publisher', 'value');
+  const edition = getFieldValue(mappingDetails, 'edition', 'value');
+  const contributors = getFieldValue(mappingDetails, 'contributors', 'subfields');
+  const productIds = getFieldValue(mappingDetails, 'productIds', 'subfields');
+  const internalNote = getFieldValue(mappingDetails, 'internalNote', 'value');
+
+  const contributorsVisibleColumns = ['contributor', 'contributorNameTypeId'];
+  const productIdsVisibleColumns = ['productId', 'qualifier', 'productIdType'];
+
+  const contributorsMapping = {
+    contributor: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.orderDetails.contributors.contributor`} />
+    ),
+    contributorNameTypeId: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.orderDetails.contributors.contributorType`} />
+    ),
+  };
+  const productIdsMapping = {
+    productId: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.orderDetails.productIdentifiers.productId`} />
+    ),
+    qualifier: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.orderDetails.productIdentifiers.qualifier`} />
+    ),
+    productIdType: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.orderDetails.productIdentifiers.productIdType`} />
+    ),
+  };
+
+  const contributorsFormatter = {
+    contributor: x => x?.contributor || noValueElement,
+    contributorNameTypeId: x => x?.contributorNameTypeId || noValueElement,
+  };
+  const productIdsFormatter = {
+    productId: x => x?.productId || noValueElement,
+    qualifier: x => x?.qualifier || noValueElement,
+    productIdType: x => x?.productIdType || noValueElement,
+  };
+
+  const contributorsFieldsMap = [
+    {
+      field: 'contributor',
+      key: 'value',
+    }, {
+      field: 'contributorNameTypeId',
+      key: 'value',
+    }
+  ];
+  const productIdsFieldsMap = [
+    {
+      field: 'productId',
+      key: 'value',
+    }, {
+      field: 'qualifier',
+      key: 'value',
+    }, {
+      field: 'productIdType',
+      key: 'value',
+    }
+  ];
+
+  const contributorsData = transformSubfieldsData(contributors, contributorsFieldsMap);
+  const productIdsData = transformSubfieldsData(productIds, productIdsFieldsMap);
+
+  const formattedSubscriptionFrom = (
+    <FormattedDate
+      value={subscriptionFrom}
+      day="2-digit"
+      month="2-digit"
+      year="numeric"
+    />
+  );
+  const formattedSubscriptionTo = (
+    <FormattedDate
+      value={subscriptionTo}
+      day="2-digit"
+      month="2-digit"
+      year="numeric"
+      format="YYYY-MM-DD"
```

Comment:
Could you please investigate can we create const `DATE_FORMAT = 'YYYY-MM-DD'; ` and reuse it across all project?